servers/lib/server/handlers/createteam.test.coffee

```diff
@@ -86,9 +86,9 @@ runTests = -> describe 'server.handlers.createteam', ->
           expect(err).to.not.exist
           expect(group.title).to.be.equal companyName
 
-          expect(group.payment).to.exist
-          expect(group.payment.subscription).to.exist
-          expect(group.payment.subscription.status).to.be.equal Status.TRIALING
+          # expect(group.payment).to.exist
```

Comment:
cc @usirin 

lets talk about on how to change the behaviour here with your changes